### PR TITLE
Works & About myself  Sections

### PR DESCRIPTION
**Pull request by LuisSalas94**
**Hello Microverse**
In this branch 
- Added images
- Added works section markup
- Added works section styles
- Added about myself section markup
- Added about myself section styles
- Implemented button interactions (enabled, hover, pressed, disabled)
- Fixed Linters issues
- Fixed Webhint issues
- Required changes done

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -54,5 +57,174 @@ <h1>I'm Fernando Glad to see <br> you!</h1>
...
+				<p class="card-description">
+					A daily selection of privately personalized reads; no accounts or
+					sign-ups required.
+				</p>
+				<div class="card-skills">
+					<ul>
+						<li><a href="#">html</a></li>
+						<li><a href="#">css</a></li>
+						<li><a href="#">javaScript</a></li>
+					</ul>
```
Comment (julie-ify) [inline index.html]: - [ ] I'm so impressed with how you used the correct tags `ul` and `li` here 🔥. However, please remove the `a` tag from the lists as the list here is just a list of technologies and not a link 😄.

File: index.html [hunk 2]
```diff
@@ -54,5 +57,174 @@ <h1>I'm Fernando Glad to see <br> you!</h1>
...
```
Comment (julie-ify) [inline index.html]: - [ ] I'm so impressed with how you used the correct tags `ul` and `li` here 🔥. However, please remove the `a` tag from the lists as the list here is just a list of technologies and not a link 😄.

File: index.html [hunk 3]
```diff
@@ -54,5 +57,174 @@ <h1>I'm Fernando Glad to see <br> you!</h1>
...
```
Comment (julie-ify) [inline index.html]: - [ ] I'm so impressed with how you used the correct tags `ul` and `li` here 🔥. However, please remove the `a` tag from the lists as the list here is just a list of technologies and not a link 😄.

File: index.html [hunk 4]
```diff
@@ -54,5 +57,174 @@ <h1>I'm Fernando Glad to see <br> you!</h1>
...
```
Comment (julie-ify) [inline index.html]: - [ ] I'm so impressed with how you used the correct tags `ul` and `li` here 🔥. However, please remove the `a` tag from the lists as the list here is just a list of technologies and not a link 😄.

File: index.html [hunk 5]
```diff
@@ -54,5 +57,174 @@ <h1>I'm Fernando Glad to see <br> you!</h1>
...
+			<h1>
+				About <br />
+				Myself
+			</h1>
+			<p>
+				Hello I’m a software developer! I can help you build a product , feature
+				or website Look through some of my work and experience! If you like what
+				you see and have a project you need coded, don’t hestiate to contact me.
+			</p>
+			<a href="#" class="connect">let's connect</a>
```
Comment (julie-ify) [inline index.html]: - [ ] Please consider replacing the `a` tag here with the `h2` tag as this text is not a link.

```suggestion
			</p>
			<h2 class="connect">let's connect</h2>
```

File: index.html [hunk 6]
```diff
@@ -54,5 +57,174 @@ <h1>I'm Fernando Glad to see <br> you!</h1>
...
+						<h4>JavaScript</h4>
+					</li>
+					<li class="skill html">
+						<img src="img/html.png" alt="html logo" />
+						<h4>HTML</h4>
+					</li>
+					<li class="skill css">
+						<img src="img/css.png" alt="css logo" />
+						<h4>CSS</h4>
+					</li>
```
Comment (julie-ify) [inline index.html]: - [ ] Good job using the `ul` and `li` tags here. However, to make the code more readable, please lets wrap the technology in Language category(e.g., "JavasScript", "HTML" and "CSS") in a list `ul` and each of them a `li`.

```suggestion
					<li class="main-skill">
						<h2>Languages</h2>
						<i class="fas fa-angle-down"></i>
						<ul>
						  <li class="skill js">
						    <img src="img/js.png" alt="javascript logo" />
						    <h4>JavaScript</h4>
					          </li>
					          
					          <li class="skill HTML">
						    <img src="img/html.png" alt="html logo" />
						    <h4>HTML</h4>
					          </li>
					         
					         <li class="skill CSS">
						    <img src="img/css.png" alt="css logo" />
						    <h4>CSS</h4>
					         </li>
					      </ul>
					</li>
```

File: style.css [hunk 7]
```diff
@@ -82,3 +126,207 @@ body {
...
+  font-size: 20px;
+  margin-right: 30px;
+}
+
+.about .main-skill h2 {
+  font-size: 18px;
+  line-height: 24px;
+  color: #000;
+  font-weight: 500;
+}
```
Comment (julie-ify) [inline style.css]: - [ ] Good job implementing these styles from Figma. However, the `font-size` on Figma design is `20px`. consider using the correct `font-size` so as to match your chosen design template.

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @LuisSalas94, 

Good job so far!

## Highlight!
- Correct Github flow ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Followed Figma design template carefully✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Kingstalux — APPROVED** [review]
Hi @LuisSalas94 , 
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

Good job implementing the required changes 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
- [ ] (OPTIONAL) The live demo link in your README  file does not take you to the right project site. It would be nice to correct that or remove the link completely.



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**